### DISCUSSION TITLE
参考文献直接都不显示问题

### DISCUSSION THREAD

**Discussion by pangxiao2013** [sjtug/SJTUThesis#1000] [提问]
**问题描述**:
check the 'main.log' for more detailsPleaseError!

**系统信息**:

- OS版本(带版本号): macOS/Windows/Linux
- TeX 发行版(带版本号): TeX Live/MiKTeX/MacTeX

**日志信息**:

**Comment by LogCreative** [comment]
建议提供一下报错信息。
